tools/build_defs/framework.bzl

```diff
@@ -87,6 +87,14 @@ CC_EXTERNAL_RULE_ATTRIBUTES = {
         allow_files = True,
         default = [],
     ),
+    "env": attr.string_dict(
+        doc = (
+            "Environment variables to set during the build. " +
+            "$(location) macros may be used to point at files which are listed as data deps, but unlike with other rules, " +
```

Comment:
For my own curiosity, is not not recommended to use something like `execpath` and `rootpath`?
https://docs.bazel.build/versions/master/be/make-variables.html#predefined_label_variables

I personally have always used `location` but don't know if that's really "legacy" behavior. Is there something "modern" we could write in the docs?

Reply:
Good call - I always forget that there are other macros! Switched to only support `execpath` because that's the thing we're really clearly actually supporting (by literally prepending the exec root!)